app/src/main/res/values/strings.xml

```diff
@@ -54,6 +54,13 @@
     <string name="notification_content">Notification Content</string>
     <string name="notification_title">Notification Title</string>
 
+    <string name="mentor_header">PROJECTS</string>
+    <string name="fav_button_text">Favorites</string>
+    <string name="spinner_default">Select Category</string>
+    <string name="web_dev_project">WEB DEV</string>
+    <string name="language_project">LANGUAGE</string>
+    <string name = "systems_project">SYSTEMS</string>
+
```

Comment:
The categories (subject to change) are "DATA SCI, LANGUAGES, SYSTEMS, and WEB DEV"
Also note that they're in alphabetical order. We want to stay consistent with iOS